script/default-site

```diff
@@ -12,7 +12,7 @@ workdir=$(pwd)
 echo "$0: creating new default site"
 if [[ "$(uname -s)" != "Linux" ]]
 then 
-  exec exe/jekyll new tmp/default-site
+   `exe/jekyll new tmp/default-site`
```

Comment:
Why did you wrap this to backticks? That matters little in a bash script, and I would prefer exec stay as we've no reason to hold our own script open rather than replace it, the latter should have also of had exec too, because we care little after we launch the script and their exit status is the status to rule them all.  In any event the backtick is entirely unnecessary.